tests/integration/sugar/test_transaction.py

```diff
@@ -59,6 +61,9 @@ async def test_none_as_destination_tag(self, client):
             destination=classic_address_to_xaddress(DESTINATION, None, False),
         )
 
+        # Silence warnings from send_reliable_submission being deprecated
```

Comment:
IMO the tests should be updated to not use `send_reliable_submission` instead of adding the silencing

Reply:
We did update most tests. Some should remain to ensure we arent breaking deprecated code.

Reply:
Its saying that something is going away so switch to using something else.